122623/css/style.css

```diff
@@ -2231,7 +2231,7 @@ background-color: #f53c48;
 }
 .wireless-card-button a {
     padding: 12px 28px;
-    background: #EB212E;
+    background: blue;
```

Comment:
Please change the ccolor of this -- #000000

Reply:
I have changed the color of button as you provide me